pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTableRestletResource.java

```diff
@@ -687,6 +688,26 @@ public RebalanceResult rebalance(
     }
   }
 
+  @GET
+  @Path("/tables/{tableName}/rebalance/status")
```

Comment:
Let's make the API more specific, and it might be better to put it under the segment rest API: `/segments/{tableName}/externalViewMismatch` and returns a map from the mismatch segment to its ideal state and external view